rfc/4/index.md

```diff
@@ -260,12 +304,36 @@ class Orientation(str, Enum):
     proximal_to_distal = "proximal-to-distal"
     # Describes the directional orientation from the periphery of an anatomical structure or limb to the center of the body.
     distal_to_proximal = "distal-to-proximal"
+
+class Orientation(BaseModel):
+    """
+    Orientation object that can represent different types of orientation information.
+    """
+    type: str
+    value: str
+
+class AnatomicalOrientation(Orientation):
+    """
+    Anatomical orientation specific implementation.
+    """
+    type: str = "anatomical"
```

Comment:
I don't *think* it was mentioned above that the "type" field should take a default value if not present?

Reply:
If the type is `AnatomicalOrientation`, then "type" has a value "anatomical".